### PR TITLE
WD : Feat api calls and re design

### PR DESCRIPTION
**Pull request by devananth**
## Added api Calls and made explore page responsive

        - added api calls for wishlist and cart services
        - made explore page responsive
        - re-designed ProductCard
        - made minor fixes in order to adapt the new changes

**Live Preview:** https://lens-store-react-1qw3hoc40-devananth.vercel.app/

### CODE REVIEW

**Review by mnath30 — APPROVED** [review]
Good work on the UI
Just a small suggestion: 1. "Remove from Cart" can be used in place of "Remove to Cart" in the Cart page
2. In the products page the button after adding to cart is shown as "Move to Cart" but I guess you meant "Go To Cart" because after clicking on that we are navigated to the Cart page

Rest all looks great

**Comment by devananth** [comment]
> Remove to Cart

Thanks, for pointing it out. Made the necessary changes

File: src/custom-hooks/useAxios.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+import { useState } from "react";
+
+const useAxios = () => {
+  const [loader, setLoader] = useState(false);
+  const [response, setResponse] = useState(null);
+  const [error, setError] = useState(null);
+
+  const callAPI = async (params) => {
+    console.log("Call Api : ", params);
```
Comment (Vishruta-Patil) [inline src/custom-hooks/useAxios.js]: Remove console.log

File: src/contexts/wishlist-context.js [hunk 2]
```diff
@@ -8,14 +10,63 @@ const initialwishlistState = {
...
+        toast.success("Added to wishlist");
+      } else if (updatedList.length < wishlist.length) {
+        toast.success("Removed from wishlist");
+      }
+      setWishlist(updatedList);
+    }
+  }, [response]);
+
+  const addToWishlistServerCall = (product) => {
+    console.log("Add wishlist : ", product);
```
Comment (Vishruta-Patil) [inline src/contexts/wishlist-context.js]: Remove console.log

**Review by Vishruta-Patil — APPROVED** [review]
Remove all the console, if it is not required. Good implementation of the useAxios hook

File: src/custom-hooks/useAxios.js (hunk 1 above)
Comment (devananth) [reply src/custom-hooks/useAxios.js]: Thanks, for pointing it out.

File: src/contexts/wishlist-context.js (hunk 2 above)
Comment (devananth) [reply src/contexts/wishlist-context.js]: Thanks, for pointing it out.

**Pull request merged**